appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/properties/YoungAndroidLegoEv3GyroSensorModeChoicePropertyEditor.java

```diff
@@ -16,10 +16,11 @@
  */
 public class YoungAndroidLegoEv3GyroSensorModeChoicePropertyEditor extends ChoicePropertyEditor {
 
-  // Lego Mindstorms EV3 sensor port choices
+
+    // Lego Mindstorms EV3 sensor port choices
   private static final Choice[] modes = new Choice[] {
-    new Choice("angle", "angle"),
-    new Choice("rate", "rate"),
+          new Choice(MESSAGES.angleValues(), "angle"),
```

Comment:
You're missing an import here for MESSAGES.

Reply:
fixed